build.sbt

```diff
@@ -527,10 +527,23 @@ lazy val runProjectDependencyGeneration =  ThisBuild / taskKey[Unit] ("Regenerat
 
 runProjectDependencyGeneration := {
   import scala.sys.process.*
+  import scala.util.{Try, Success, Failure}
+
   val s: TaskStreams = streams.value
-  val uid = "id -u".!!.stripSuffix("\n")
-  val gid = "id -g".!!.stripSuffix("\n")
-  val baseCommand = s"docker run --userns=host --rm -u $uid:$gid -v ${baseDirectory.value.getAbsolutePath}/:/data minlag/mermaid-cli -i OPAL/ProjectDependencies.mmd -c mermaid-config.json"
+
+  val dockerUserArg = Try {
+      val uid = "id -u".!!.stripSuffix("\n").toString
+      val gid = "id -g".!!.stripSuffix("\n").toString
+      s"$uid:$gid"
+  } match {
+    case Success(userSpec) =>
+      s"-u $userSpec"
+    case Failure(ex) =>
+        s.log.warn(s"Cannot obtain user id, using mermaid default user for diagram file creation")
+      ""
+  }
```

Comment:
Wouldn't it be easier to do
```
val dockerUserArg = Try {
      val uid = "id -u".!!.stripSuffix("\n").toString
      val gid = "id -g".!!.stripSuffix("\n").toString
      s"-u $uid:$gid"
  }.getOrElse("")
```
? I don't see a need for the warning on Windows and on Linux, if the Try fails, the user will get a docker error anyway.